app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i.freeze
```

Comment:
- [ ] **[OPTIONAL]** You could use the built in standard library for email regex `URI::MailTo::EMAIL_REGEXP`

Reply:
Thank you for the feedback. We have added this change. Thanks!